packages/type-jsonrpc/src/types.ts

```diff
@@ -2,14 +2,14 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import { CodecTypes } from '@polkadot/types/classes';
+// import { CodecTypes } from '@polkadot/types/classes';
 
 type PubSub = [string, string, string];
 
 export interface RpcParam {
   isOptional: boolean;
   name: string;
-  type: CodecTypes;
+  type: string; // CodecTypes;
```

Comment:
Yes. Basically at this point need to add more `@ts-igore` lines that actual code. However... believe this is solvable now in the same was as `ClassOf` (some work required, but solvable)